apps/server/src/tests/auth.test.ts

```diff
@@ -0,0 +1,78 @@
+import { client, db } from '../clients/postgres';
+import { Users } from '../db/schema';
+import { httpServer } from '../index';
+import request from 'supertest';
+import { eq } from 'drizzle-orm';
+import { generateEncodedString } from '../utilities/generateEncodedString';
+
+const { JWT_AUTH_SECRET_KEY, JWT_AUTH_EXPIRES_IN } = process.env;
+
+const server = request(httpServer);
+
+interface IUserRegisterObject {
```

Comment:
You should use your zod validator schema type inference for this not to create duplicate types. 
`type IUserRegisterObject = z.infer<typeof userValidatior>` You should create this type inside your validators and export it so that you will be able to use it application wide

Reply:
Continues